vcs.mk

```diff
@@ -11,34 +11,56 @@ endif
 CLOCK_PERIOD ?= 1.0
 RESET_DELAY ?= 777.7
 
+#----------------------------------------------------------------------------------------
+# gcc configuration/optimization
+#----------------------------------------------------------------------------------------
+# -flto slows down compilation on small-memory and breaks on firesim-manager
+CMODE := -O3 -fbranch-probabilities -march=native
+
+VCS_CXXFLAGS = \
+	$(CXXFLAGS) \
+	$(CMODE) \
+	-I$(RISCV)/include \
+	-I$(dramsim_dir) \
+	-std=c++11 \
+	$(EXTRA_SIM_CXXFLAGS)
+
+VCS_LDFLAGS =	\
+	$(LDFLAGS) \
+	-L$(RISCV)/lib \
+	-Wl,-rpath,$(RISCV)/lib \
+	-L$(sim_dir) \
```

Comment:
There is getting to be an awful lot of overlap between these sets of flags for verilator and vcs, but I think we might want to wait for a different PR since we are already changing so much.

Reply:
Yes. That was partly the intention of doing this... since if we can consolidate flags then this might be easier to deal with in the long run.